src/main/java/pl/lodz/p/it/zzpj/botsite/web/controllers/MessageAdminController.java

```diff
@@ -71,7 +72,7 @@ public MessageDTO getUsersMessageById(@PathVariable("userId") final Long userId,
     )
     @ResponseStatus(HttpStatus.OK)
     public MessageDTO editMessageForUser(@PathVariable("userId") Long userId, @PathVariable("messageId") Long messageId,
```

Comment:
Nie rozumiem tej całej idei "...ForUser". Czym to się wyróżnia od zwykłej edycji dla admina?

Reply:
nazwą